src/constants.rs

```diff
@@ -61,4 +61,73 @@ pub const WEATHER_CODES: &[(i32, &str)] = &[
     (431, "🌨️"),
 ];
 
+pub const WEATHER_CODES_NERD: &[(i32, &str)] = &[
+    (113, ""),
+    (116, ""),
+    (119, ""),
+    (122, ""),
+    (143, ""),
```

Comment:
Some of the characters are removed in the newest version of Nerd Font, please consider update them.